src/app/api/postTerms.ts

```diff
@@ -0,0 +1,28 @@
+import { apiWrapper } from "./apiWrapper";
+
+interface TermsPayload {
+  address: string;
+  termsAccepted: boolean;
+  publicKey: string;
+}
+
+export const postTerms = async (
+  address: string,
+  termsAccepted: boolean,
+  publicKey: string,
+) => {
+  const payload: TermsPayload = {
+    address: address,
+    termsAccepted: termsAccepted,
```

Comment:
Is there a possibility of this ever being `false`?

Reply:
agree, removed the param